app/src/main/java/com/pedromassango/doubleclick/DoubleClick.java

```diff
@@ -45,6 +45,12 @@
    */
   public DoubleClick(final DoubleClickListener doubleClickListener) {
     this.doubleClickListener = doubleClickListener;
```

Comment:
Instead of initializing those variables manually, maybe this constructor should make a call to the other one:
```suggestion
    this(doubleClickListener, 200L);
```